README.md

```diff
@@ -82,28 +74,69 @@ ansible-galaxy collection build --force
 ansible-galaxy collection install cisco-aci-* --force
 ```
 
-### Second Approach
+## Use Cases
+
+Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+
+### Adding a new EPG
 
-Go to [ansible-aci Actions](https://github.com/CiscoDevNet/ansible-aci/actions)
+```yml
+- hosts: aci
+  gather_facts: no
 
-Select the latest CI build
+  tasks:
+  - name: Add a new EPG
+    cisco.aci.aci_epg:
+      hostname: apic
+      username: admin
+      password: SomeSecretPassword
+      tenant: production
+      ap: intranet
+      epg: web_epg
+      description: Web Intranet EPG
+      bd: prod_bd
+    delegate_to: localhost
+```
 
-Under Artifacts download collection and unzip it using Terminal or Console.
+## Optimizing Playbooks
 
-*Note: The collection file is a zip file containing a tar.gz file. We recommend using CLI because some GUI-based unarchiver might unarchive both nested archives in one go.*
+There are two main methods to optimize the execution of ACI modules in your playbooks.
 
-Install the unarchived tar.gz file
+1. Using the ACI HTTPAPI plugin
+1. Using the `suppress_` options
 
-```sh
-ansible-galaxy collection install cisco-aci-1.0.0.tar.gz —-force
-```
```

Comment:
Why removing this second approach?

Reply:
@akinross and I agreed, that since the builds aren't consistently green due to the integration tests that we wouldn't recommend doing it this way anymore.
https://github.com/CiscoDevNet/ansible-aci/pull/692#discussion_r1799281736
The idea was to add an nightly dev build that gets published instead. Not sure if this is possible with Ansible Galaxy but thought I would add a backlog item for it.

Reply:
How would the nightly be different from the CI actions downloads and how is it different from building from source in regard to the integration tests?

How are we tracking this nightly suggestion?

Until we have the nightly available more easily I would keep that in.

Reply:
Its a good point, there would not be much difference. The nightly build would just be pushing the latest master which gets built in the CI every day anyway.

I have added the second approach back in.